### ISSUE TITLE
Outer bags

### ISSUE THREAD

**Issue by jesterhui** [interpretml/interpret#109]
First, I would like to say that I am greatly appreciative of this release. One question—in a footnote in the arxiv'd paper 'InterpretML: A Unified Framework for Machine Learning Interpretability', you recommend "using reference parameters: 100 inner bags, 100 outer bags, 5000 epochs, and a learning rate of 0.01" for best accuracy and interpretability with regards to constructing EBM models. I figure that feature_step_n_inner_bags is inner bags, data_n_episodes is epochs, but it eludes me which attribute specifies the outer bags? Thanks!

**Comment by interpret-ml** [comment]
Hi @jesterhui , 

Glad to hear you enjoyed the package and the arxiv paper! Thanks for the feedback -- we're planning on updating the parameter names in an upcoming release to make them more clear (please note that this would be a breaking change). 

In the meanwhile, here's a translation guide for those paramters:

Inner Bags: "feature_step_n_inner_bags"
Outer Bags: "n_estimators" # This is also what gets parallelized when n_jobs != 1
Epochs: "data_n_episodes"

If you're interested in training pairwise interaction terms as well, set the "interactions" parameter to either a number "n" to automatically detect and train the "n" most significant interaction terms, or list of lists (i.e. [[2,3], [1,3]]) to train specific interactions. A small number of interaction terms can often improve performance on some datasets.

Let us know if you have any further questions!

-InterpretML Team
